### PR TITLE
Modified JavaScript Batch Samples to run with updated storage library and VM image sku

### PR DESCRIPTION
**Pull request by NickKouds**
Modified the JavaScript Batch samples to be runnable with the updated python storage library and valid Node sku as part of an underlying effort to update JavaScript Batch documentation from the deprecated Node.JS SDK to the JavasScript/TypeScript SDK.

### CODE REVIEW

File: JavaScript/Node.js/processcsv.py [hunk 1]
```diff
@@ -60,22 +60,32 @@ def processcsvfile(fname,seperator,outdir,outfname):
...
+            with open(json_outpath, "rb") as data:
+                container_client.upload_blob(name=str(pattern + 'json/' + blob_name + ".json"), data=data)
         else:
             json_outpath = processcsvfile(fname=downloadedblob,seperator="|",outfname=blob_name,outdir='jsonfiles/' + container + "/")
-            print("uploading blob" + json_outpath)
-            block_blob_service.create_blob_from_path(container_name=container,blob_name=str('json/' + blob_name + ".json"),file_path=json_outpath)            
\ No newline at end of file
+            print("uploading blob " + json_outpath)
+            with open(json_outpath, "rb") as data:
+                container_client.upload_blob(name=str('json/' + blob_name + ".json"), data=data)
```
Comment (paterasMSFT) [inline JavaScript/Node.js/processcsv.py]: Seems like most of this is being repeated in both conditions.

**Review by paterasMSFT — APPROVED** [review]
Looks great! Just one suggestion (see comment).

**Review by wiboris — APPROVED** [review]
looks good

File: JavaScript/Node.js/processcsv.py (hunk 1 above)
Comment (NickKouds) [reply JavaScript/Node.js/processcsv.py]: Good catch! Will consolidate that logic.

**Pull request merged**